src/util/FactorUtil.js

```diff
@@ -32,18 +32,12 @@ function (Okta, TimeUtil) {
       label: 'factor.oktaVerifyPush',
       description: 'factor.push.description',
       iconClassName: 'mfa-okta-verify',
-      sortOrder: 1
-    },
-    'SMS': {
-      label: 'factor.sms',
-      description: 'factor.sms.description',
-      iconClassName: 'mfa-okta-sms',
       sortOrder: 2
```

Comment:
shouldn't this be the same as ```OKTA_VERIFY```? They are at the end the same factor

Reply:
Fixed